src/components/SelectionList/Search/ActionCell.tsx

```diff
@@ -41,15 +50,31 @@ function ActionCell({
     isChildListItem = false,
     parentAction = '',
     isLoading = false,
+    policyID = '',
+    reportID = '',
 }: ActionCellProps) {
     const {translate} = useLocalize();
     const theme = useTheme();
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
     const {isOffline} = useNetwork();
 
+    const [iouReport] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {canBeMissing: true});
     const text = isChildListItem ? translate(actionTranslationsMap[CONST.SEARCH.ACTION_TYPES.VIEW]) : translate(actionTranslationsMap[action]);
     const shouldUseViewAction = action === CONST.SEARCH.ACTION_TYPES.VIEW || (parentAction === CONST.SEARCH.ACTION_TYPES.PAID && action === CONST.SEARCH.ACTION_TYPES.PAID);
+    const [chatReport] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${iouReport?.chatReportID}`, {canBeMissing: true});
+    const bankAccountRoute = getBankAccountRoute(chatReport);
+
+    const confirmPayment = useCallback(
+        (type: PaymentMethodType | undefined) => {
+            if (!type || !chatReport) {
+                return;
+            }
+
+            payMoneyRequest(type, chatReport, iouReport);
```

Comment:
@getusha I think we might need to use `payMoneyRequestOnSearch` here? Like we have been using it, Any thoughts? 👀